pkgs/by-name/go/gogogadget/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "gogogadget";
+  version = "0.0.1-unstable-2023-03-30";
```

Comment:
Let's see if we can get a release in a timely manner.

https://github.com/VigilantSys/GoGoGadget/issues/14